uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/request-ctrl.js

```diff
@@ -968,15 +942,21 @@ define([
       }
 
       /**
+       * Initialize user's role to either owner, manager or admin
+       */
+      function initUserRole () {
+        role = this.directiveOptions.userRole;
```

Comment:
I tried that, but `this`does not have child directives values when role is defined. 
`directiveOptions`is set after calling parent controller. That is why moved the same to a common place which is called by the child, i.e. `_initRequestAttributes `